### PR TITLE
Added reducers, actions, imported the store and provider for redux fo…

### PR DESCRIPTION
**Pull request by cn8817**
What I did:
- Added reducers and actions for the parent component to help manage state
- Added the store and provider into index.js to manage the state

What files were changes:
- src/redux/actions/parentActions
- src/redux/reducers/parentReducer
- src/redux/reducers/index.js
- src/index.js

Why were these changes made:
- To successfully have a state management system in react using the redux store

Coauthors @JoeyMBrown & @MikeLikesCode 

!(https://www.loom.com/share/fa87e9052e0e40b5b65184436ede59f7)

### CODE REVIEW

**Comment by bradlylewis** [comment]
Wow you guys really did a lot!

**Review by bradlylewis — APPROVED** [review]

File: src/redux/actions/parentActions.js [hunk 1]
```diff
@@ -1 +1,65 @@
...
+    .catch(err => {
+      dispatch({ type: ERROR_ACTION, payload: err });
+    });
+};
+
+export const getInbox = dispatch => {
+  dispatch({ type: GET_INBOX_ACTION });
+  axios
+    .get('https://dummyapi.io/data/v1/user/60d0fe4f5311236168a109ca', {
+      crossdomain: true,
```
Comment (fernando817mm) [inline src/redux/actions/parentActions.js:57]: Great job but can you explain what the purpose of line 57 is? Is it a cors error solution?

**Review by fernando817mm — APPROVED** [review]

**Review by Jie-chelchel — APPROVED** [review]
Looks good!

File: src/redux/actions/parentActions.js (hunk 1 above)
Comment (MikeLikesCode) [inline src/redux/actions/parentActions.js:57]: We could probably get rid of the crossdomain header here. This can be done next time when we add our api.

**Review by MikeLikesCode — APPROVED** [review]

**Review by Rodgers31 — APPROVED** [review]
Looks great

**Pull request merged**